packages/devtools_app/test/test_infra/test_data/memory/heap/heap_graph_fakes.dart

```diff
@@ -6,54 +6,36 @@ import 'dart:typed_data';
 
 import 'package:collection/collection.dart';
 import 'package:devtools_app/src/shared/memory/class_name.dart';
+import 'package:devtools_app/src/shared/memory/simple_items.dart';
+import 'package:flutter_test/flutter_test.dart';
 import 'package:vm_service/vm_service.dart';
 
 typedef RefsByIndex = Map<int, List<int>>;
 typedef ClassByIndex = Map<int, HeapClassName>;
 
-class HeapSnapshotGraphMock implements HeapSnapshotGraph {
-  HeapSnapshotGraphMock();
+final _sentinelObject = _HeapSnapshotObjectFake();
 
-  @override
-  int get capacity => throw UnimplementedError();
+const int _defaultClassId = 0;
+const int _weakClassId = 1;
+
+class HeapSnapshotGraphFake extends Fake implements HeapSnapshotGraph {
```

Comment:
naming nit: rename to FakeHeapSnapshotGraph to match naming convention for fakes. Same goes for all of these fakes below